webapp/channels/src/i18n/en.json

```diff
@@ -4584,6 +4584,7 @@
   "post_info.tooltip.add_reactions": "Add Reaction",
   "post_info.unpin": "Unpin from Channel",
   "post_info.unread": "Mark as Unread",
+  "post_limits_announcement_bar.copyText": "Message limits exceeded. Contact administrator with: ERROR_SAFETY_LIMITS_EXCEEDED",
```

Comment:
Nit: Wonder if we should have the "error code" out of the translation string.

We will never translate it, and if we ever change it (I don't think that we will, but still...) it we will have to change it in every language.

But 0/5 on this.

Reply:
Makes sense, done!